embed/vite.config.ts

```diff
@@ -5,7 +5,21 @@ export default defineConfig(({ mode }) => {
   const isDevelopment = mode === 'development'
 
   return {
-    envDir: resolve(__dirname),
+    define: {
+      'import.meta.env.VITE_SCRIPT_FRONTEND_URL': JSON.stringify(
+        isDevelopment
+          ? 'http://localhost:3000'
```

Comment:
Might be needed how it's used.
```suggestion
          ? 'http://localhost:3000/'
```